lib/components/util/link.tsx

```diff
@@ -31,4 +31,9 @@ const Link = ({
   </RouterLink>
 )
 
-export default Link
+const DummyLink = ({ children, to }: Props): JSX.Element => (
+  // Do not render router links in snapshots.
```

Comment:
Why do we want to avoid this?

Reply:
Yeah I need to update the comment. you would have to setup an entire stack of providers, routes, and switches in the tests, and I didn't find an easy way to do it.

Reply:
The alternative otherwise is to just render an anchor `<a>` with the desired `href`.

Reply:
Thoughts, @miles-grant-ibigroup?

Reply:
Why are we not using anchor tags?